src/controllers/SicSearchController.ts

```diff
@@ -19,7 +18,7 @@ export class SicSearchController {
     });
   };
 
-  public search = async (req: Request, res: Response, next: NextFunction) => {
+  public search = async (req: Request, res: Response) => {
     const matchOptions = req.body.matchOptions ?? "or";
     let matchPhrase = false;
     if (matchOptions === "and") {
```

Comment:
Is this being tested? `matchOptions === "and"` 

Code coverage is reporting that this branch and the scenario an error is caught isn't being tested